README.md

```diff
@@ -448,9 +448,9 @@ referred to be name using `Ref`, and utilized via `Apply`.
 
 List of builtin functions:
 
-| Function                       | Description                                                                    | Example Usage                    |
-|--------------------------------|--------------------------------------------------------------------------------|----------------------------------|
-| `+, -, *, //, /, >, <, >=, <=` | Standard Math                                                                  |                                  |
+| Function                       | Description                                                                  | Example Usage                    |
+|--------------------------------|------------------------------------------------------------------------------|----------------------------------|
+| `+, -, *, //, /, >, <, >=, <=` | Standard Math                                                                |                                  |
```

Comment:
nit: Looks like this needs some more alignment correction.

Reply:
e78dddb